src/CommonITILObject.php

```diff
@@ -2557,6 +2557,18 @@ public function post_addItem()
         $this->manageValidationAdd($this->input);
 
         parent::post_addItem();
+        
+        if (isset($_SESSION['glpiis_ids_visible']) && !$_SESSION['glpiis_ids_visible']) {
+            Session::addMessageAfterRedirect(sprintf(
+                __('%1$s: %2$s'),
+                __('Your ticket has been registered.'),
```

Comment:
Indeed message should be reviewed. As it is already prefixed by `$this->getTypeName(1)`, maybe a generic text could be used here.

Reply:
Maybe "Your request has been registered"?

Or maybe it would be better to modify CommonDBTM::addMessageOnAddAction()
https://github.com/glpi-project/glpi/blob/a34fb7f8645acf1ab4de99420bc4f96734b35424/src/CommonDBTM.php#L1472-L1473

And modify it like:
```
$opt = [ 'forceid' => $this instanceof CommonITILObject ];
$display = (isset($this->input['_no_message_link']) ? $this->getNameID($opt)
                                                            : $this->getLink($opt));
```
And remove additional code from Ticket?

Reply:
Submitted another PR #12199 with changes in CommonDBTM